src/map/npc.cpp

```diff
@@ -5089,6 +5089,17 @@ static const char* npc_parse_mob(char* w1, char* w2, char* w3, char* w4, const c
 		return strchr(start,'\n');// skip and continue
 	}
 
+	// Check if sprite is the mob name or ID
+	char *pid;
```

Comment:
```suggestion
	char *pid;
	sprite[NAME_LENGTH-1] = '\0';
```
sprite is uninitialized and could therefore be not zero terminated.